### ISSUE TITLE
The loss does not decrease

### ISSUE THREAD

**Issue by oysz2016** [WenmuZhou/PAN.pytorch#3]
It's a great job.Before your update, I tried to train PAN, but the loss was still high until the end of the training.Does the current version support effective training and inference?

**Comment by lgcy** [comment]
i meet the problem too, the loss is non-convergence

**Comment by ghost** [comment]
@oysz2016 @lgcy 
- Which **dataset** you trained on?
- How many **epochs** & **hours** did you train for?
- What was your best your model_best **loss**?
- did you modify config.py?
- upload log?

**Comment by lgcy** [comment]
@deepseek i do not modify the config.py and  trained  on the icdar2015 for 600 epoch. Then, i got the model_best loss is 0.6

**Comment by WenmuZhou** [comment]
At present, the loss that can be achieved with the network in the pan paper is about 0.5. Change the [`get_model`](https://github.com/WenmuZhou/PAN.pytorch/blob/master/train.py#L22) to get_model_pse1 can get better results, its still in the experiment

**Comment by ghost** [comment]
@WenmuZhou I see that you listed the backbone as [resnet18](https://github.com/WenmuZhou/PAN.pytorch/blob/master/config.json#L35), did the detection rate improve?

**Comment by WenmuZhou** [comment]
resnet 18 is used on paper, resnet50 is used in my experiment

**Comment by ustczhouyu** [comment]
@WenmuZhou  作者好，感谢分享代码，请问这个复现的代码有使用PA聚类重建的吗？还是只使用pse重建？
